ARCHITECTURE.md

```diff
@@ -107,6 +107,21 @@ This table keeps track of the manifests of the validators.
 | `seq`                |The sequence number of this manifest.                       |
 
 
+### `ballot`
+
+This table keeps track of the most current voting data for the validators.
+
+| Key                  | Definition                                                        |
+|----------------------|-------------------------------------------------------------------|
+| `master_key`         |The master key of the validator.                                   |
+| `signing_key`        |The signing key of the validator.                                  |
+| `ledger_index`       |The most recent ledger index where voting data was retrieved.      |
+| `amendments`         |The amendments this validator wants to be added to the protocol.   |
```

Comment:
This might make sense to join another table to make it easier to query an amendment and get the list of validators voting on it.

Reply:
Yes we will need to join with the `validators` table for both `validators` and `amendments` APIs (for the second API we will need `networks` data from `validators` table)

Reply:
Will it be easier to do that with amendments just being an array of comma separated fields?